openmdao/approximation_schemes/approximation_scheme.py

```diff
@@ -296,6 +300,8 @@ def _init_approximations(self, system):
     def _compute_approximations(self, system, jac, total, under_cs):
         from openmdao.core.component import Component
 
+        self._progress_out = self.out_stream if hasattr(self, "out_stream") else None
```

Comment:
Again cut out the middleman.  Put the out_stream in self._progress_out, as that is what it is for.